manifests/s/Spotify/Spotify/1.1.72.439.gc253025e/Spotify.Spotify.installer.yaml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 PackageIdentifier: Spotify.Spotify
-PackageVersion: 1.1.71.560.gc21c3367
+PackageVersion: 1.1.72.439.gc253025e
```

Comment:
I'm getting version 1.1.71.560.gc21c3367, also the hash didn't change so guess its correct

Reply:
Yeah, kind of figured it downloaded the latest version on start. Oh well.

Reply:
Looks like it took them a minute.